lib/node_modules/@stdlib/array/zeros/docs/types/index.d.ts

```diff
@@ -208,7 +52,7 @@ declare function zeros( length: number, dtype: 'generic' ): Array<number>;
 * var arr = zeros( 2, 'float32' );
 * // returns <Float32Array>[ 0.0, 0.0 ]
 */
-declare function zeros( length: number, dtype?: DataType ): Float64Array;
+declare function zeros<T extends keyof DataTypeMap<number> = 'float64'>( length: number, dtype?: T ): DataTypeMap<number>[T];
```

Comment:
This should be `NumericAndGenericDataTypeMap`.

Reply:
Changed.